deltachat-ios/Controller/Settings/AdvancedViewController.swift

```diff
@@ -219,20 +219,32 @@ internal final class AdvancedViewController: UITableViewController, ProgressAler
             headerTitle: String.localized("pref_experimental_features"),
             footerTitle: nil,
             cells: [videoChatInstanceCell, broadcastListsCell, locationStreamingCell])
-        let appAccessSection = SectionConfigs(
-            headerTitle: String.localized("pref_app_access"),
-            footerTitle: String.localized("pref_show_system_contacts_explain"),
-            cells: [showSystemContactsCell])
-        let autocryptSection = SectionConfigs(
-            headerTitle: String.localized("pref_encryption"),
-            footerTitle: nil,
-            cells: [autocryptPreferencesCell, manageKeysCell, sendAutocryptMessageCell]
-        )
-        let serverSection = SectionConfigs(
-            headerTitle: String.localized("pref_server"),
-            footerTitle: String.localized("pref_only_fetch_mvbox_explain"),
-            cells: [accountSettingsCell, sentboxWatchCell, sendCopyToSelfCell, mvboxMoveCell, onlyFetchMvboxCell])
-        return [viewLogSection, experimentalSection, appAccessSection, autocryptSection, serverSection]
+
+        if dcContext.isChatmail {
+            let encryptionSection = SectionConfigs(
+                headerTitle: String.localized("pref_encryption"),
+                footerTitle: nil,
```

Comment:
We could think about adding a custom `init` for `SectionConfigs` with `nil` as default value for `footerTitle` 🤔

Reply:
yes, maybe that makes sense. however, the pattern is used also in many other views, so, i'll stay with it for now, it is easy enough